src/Data/Avro/Deriving.hs

```diff
@@ -185,7 +188,7 @@ mkStrictPrimitiveField _ field =
 -- Person { firstName :: Text }
 -- @
 -- You may want to enable 'DuplicateRecordFields' if you want to use this method.
-mkAsIsFieldName :: TypeName -> Field -> Text
+mkAsIsFieldName :: Text -> Field -> Text
```

Comment:
Why the change to a less specific type?

Reply:
For consistency. All the other functions take `Text`, and the `DerivingOptions` expects `Text -> Field -> Text` and not `TypeName -> Field -> Text`. Maybe this one was just forgotten at some point.